packages/backend/src/apiV1.ts

```diff
@@ -261,3 +262,58 @@ apiV1Router.get('/status', isAuthenticated, async (req, res, next) => {
         })
         .catch(next);
 });
+
+apiV1Router.get('/spaces', isAuthenticated, async (req, res, next) => {
```

Comment:
Since we only have 1 space we could have avoided the FE but since it is already done I don't see why we should remove this code now. The UI and endpoints are ready to support multiple spaces.